### PR TITLE
Add content to yml

### PR DESCRIPTION
**Pull request by pmgrove**
:v:

/cc @zendesk/vegemite, @zendesk/i18n 

### Description
Adding marketplace listing to `en.yml` file.

### References
* JIRA: n/a 

### Risks
* [low] content does not show on marketplace for app

### CODE REVIEW

File: translations/en.yml [hunk 1]
```diff
@@ -6,14 +6,55 @@ packages:
...
+
+
+By enabling this app, You agree to the [Built by Zendesk Terms of Use](https://www.zendesk.com/company/built-by-zendesk-agmt/)."
+  - translation:
+      key: "txt.apps.answer_suggestion_app.app.installation_instructions"
+      title: "Installation Instructions for App: Answer Suggestion"
+      value: "To use the Answer Suggestions app, you must:
+
+
+* Be using Zendesk Help Center or Web Portal
```
Comment (injuhwang) [inline translations/en.yml]: Is "Web Portal" still valid?

Comment (pmgrove) [inline translations/en.yml]: Good point, I will remove it.

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍 for the strings

File: translations/en.json [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
...": "To use the Answer Suggestions app, you must:\n\n* Be using Zendesk Guide\n\nTo install the Answer Suggestions app:\n\n1. Sign in to your Zendesk Support account, then click the Admin icon in the sidebar.\n2. Select Apps > Marketplace,  then find the Answer Suggestions app and click the tile.\n3. On the Answer Suggestions app page, click Install app in the upper-right corner.\n4. Configure the app settings, then click Install:\n* Number of results: Set the number of results you want the app to display in the ticket sidebar. By default, the app displays ten. The more results you display, the less relevant the ones at the bottom will be. We recommend a number between four and ten.\n* Customer domain: Fill in the customer domain field only if you have a host mapped Help Center. For example, your URL is support.flylo.com instead of flylo.zendesk.com. Enter the complete URL (including http or https). For example, https://support.flylo.com/.\n\n**NOTE: Make sure you enter the complete URL, including the / at the end, if you are using this customer domain setting. **\n* Copy topic title: Check the copy topic title box if you want the app to copy both the link and title of the article."
```
Comment (injuhwang) [inline translations/en.json]: Please fix the formatting of the last three lines: https://zendesk.box.com/s/go54mkf0yj651cn8iw3ro3zoxfwhifh0

Comment (pmgrove) [inline translations/en.json]: @injuhwang, I cannot access Box, can you send via Google Drive?

Comment (injuhwang) [inline translations/en.json]: Let me know if you can't access this, either: https://goo.gl/photos/K89nvmD4HrToLYUFA

File: translations/en.yml [hunk 3]
```diff
@@ -6,14 +6,55 @@ packages:
...
-      title: "App description"
-      value: "Answer Suggestions from your knowledge base."
-  - translation:
       key: "txt.apps.answer_suggestion_app.app.name"
       title: "Name of the App"
       value: "Answer Suggestion"
   - translation:
+      key: "txt.apps.answer_suggestion_app.app.short_description"
+      title: "App description"
+      value: "Answer Suggestions from your knowledge base."
```
Comment (injuhwang) [inline translations/en.yml]: Please delete `.` at the end.

**Comment by pmgrove** [comment]
@injuhwang 💇

**Comment by injuhwang** [comment]
👍

**Pull request merged**